.github/workflows/run-scenarios.yaml

```diff
@@ -2,6 +2,12 @@ name: Run Scenarios
 on:
   workflow_dispatch:
   pull_request:
+permissions:
```

Comment:
why do we need added permissions for this action?

Reply:
@kevincheng96

Reply:
@manoj9april was getting [this error](https://github.com/dorny/test-reporter/issues/168) when running scenarios in his own forked repo, so this was the way to address that. We don't need the explicit permissions in the `comet` repo, but other teams that are developing it in their own repos might.

Reply:
cool will remove from this PR then